modules/apps/27-interchain-accounts/controller/keeper/msg_server.go

```diff
@@ -39,7 +40,12 @@ func (s msgServer) RegisterInterchainAccount(goCtx context.Context, msg *types.M
 
 	s.SetMiddlewareDisabled(ctx, portID, msg.ConnectionId)
 
-	channelID, err := s.registerInterchainAccount(ctx, msg.ConnectionId, portID, msg.Version, msg.Order)
+	// use ORDER_ORDERED as default in case msg's ordering is NONE
```

Comment:
we should probs document this yea? Would assume in versioned docs for 8.1

Reply:
I think it's already mentioned in the docs that the default will be `ORDERED`, but I will check and add it if needed (in a separate PR, since the release branch doesn't have the docs).

Reply:
for memes only :D 

```go
var order channeltypes.Order
switch msg.Order {
    case channeltypes.NONE:
        order = channeltypes.ORDERED
    default:
        order = msg.Order
}
```

resolving this comment! :)

Reply:
unsure how this comment ended up on the wrong thread 😕

Reply:
meme'd too hard you hallucinated